src/Common/Simulation.cpp

```diff
@@ -2171,11 +2175,11 @@ void Simulation<ndim>::ComputeBlockTimesteps(void)
   }
   if (nbody != NULL) {
     for (i=0; i<nbody->Nnbody; i++) {
-      assert(nbody->nbodydata[i]->level <= level_max);
+      assert(nbody->nbodydata[i]->level <= level_max); 
       assert(nbody->nbodydata[i]->nlast <= n);
-      assert(nbody->nbodydata[i]->nstep == pow(2,level_step - nbody->nbodydata[i]->level));
+      assert(nbody->nbodydata[i]->nstep >= pow(2,level_step - nbody->nbodydata[i]->level)); //comment 5: what failed then, was solved by adding inequality sign instead of equality
```

Comment:
The original assertion (nstep == pow(2, ...) ) was correct.  If it's failing, then there is a problem somewhere.